### PR TITLE
feat: add repo badges

### PR DESCRIPTION
**Pull request by darrenvechain**
# Description

Add repo badges. See example here: https://github.com/darrenvechain/thor/tree/add-repo-badges

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] New and existing E2E tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have not added any vulnerable dependencies to my code

### CODE REVIEW

**Review by nwbrettski — APPROVED** [review]
LGTM

**Review by leszek-vechain — APPROVED** [review]

**Comment by otherview** [comment]
Can the "\<a\>" be fixed ? Example in the img

image
image

**Comment by darrenvechain** [comment]
> Can the "<a>" be fixed ? Example in the img
> 
> image image

Yepp, it's fixed now

**Review by otherview — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -37,8 +62,11 @@ The Vechain Improvement Proposals (VIPs) repository can be found [here](https://
 
 To chat with other community members you can join:
 
-- [Telegram](https://t.me/VechainDevCommunity)
-- [Discord](https://discord.gg/vechain)
+<p>
+    <a href="https://discord.gg/vechain"><img src="https://img.shields.io/badge/Discord-5865F2?style=for-the-badge&logo=discord&logoColor=white" /></a>
+    <a href="https://t.me/VechainDevCommunity"><img src="https://img.shields.io/badge/Telegram-2CA5E0?style=for-the-badge&logo=telegram&logoColor=white" /></a>
```
Comment (libotony) [inline README.md:67]: We had migrated to the new telegram, which contains a new dev chat channel https://t.me/vechainandfriends, do we need to promote this?

File: docs/hosting-a-node.md [hunk 2]
```diff
@@ -6,18 +6,19 @@ requirements may change as the network evolves._
...
-  - [Authority Nodes](#authority-nodes)
-  - [Public Nodes](#public-nodes)
-- [Important Considerations](#important-considerations)
-  - [Archive Node](#archive-node)
-  - [Full Node](#full-node)
-  - [Full Node without Logs](#full-node-without-logs)
+    - [Authority Nodes](#authority-nodes)
+    - [Public Nodes](#public-nodes)
+- [Node Types](#node-types)
+    - [Archive Node](#full-archive-node)
```
Comment (libotony) [inline docs/hosting-a-node.md:12]:
```suggestion
    - [Full Archive Node](#full-archive-node)
```

File: docs/hosting-a-node.md [hunk 3]
```diff
@@ -6,18 +6,19 @@ requirements may change as the network evolves._
...
+    - [Full Node](#full-node)
+    - [Full Node without Logs](#full-node-without-logs)
 
 ___
 
 ### Command Line Options
 
-Please refer to [Command Line Options](./usage.md#command-line-options) in the usage documentation to see a list of all available options.
+Please refer to [Command Line Options](./usage.md#command-line-options) in the usage documentation to see a list of all
+available options.
```
Comment (libotony) [inline docs/hosting-a-node.md:21]: Check the line break

File: README.md [hunk 4]
```diff
@@ -9,13 +9,38 @@
...
+
+---
+
 ## Getting Started
 
 VechainThor is the layer 1 blockchain, highly compatible with EVM<sup>*</sup>, which powers the vechain ecosystem.
-VechainThor is a public blockchain that is designed for the mass adoption of blockchain technology by enterprise users of
+VechainThor is a public blockchain that is designed for the mass adoption of blockchain technology by enterprise users
+of
 all sizes and is intended to serve as a foundation for a sustainable and scalable enterprise blockchain ecosystem.
```
Comment (libotony) [inline README.md:39]:
```suggestion
VechainThor is a public blockchain that is designed for the mass adoption of blockchain technology by enterprise users
of all sizes and is intended to serve as a foundation for a sustainable and scalable enterprise blockchain ecosystem.
```

**Review by libotony — APPROVED** [review]

**Pull request merged**